Ted-T47.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -2,13 +2,7 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
You can ignore this file and the following file. I've gone through several pains to make git not track them, but I don't have time to make it properly not track them. Annoying. This is why I don't use Xcode much. It adds all these complications.